rocksdb_admin/admin_handler.cpp

```diff
@@ -377,6 +393,24 @@ bool AdminHandler::writeMetaData(
   return s.ok();
 }
 
+bool DeserializeKafkaPayload(
+    const string& payload,
+    KafkaOperationCode& op_code,
+    rocksdb::Slice& value) {
```

Comment:
for output parameter, we usually use pointers instead of references. This will make it more explicit that the function can change the parameters.

Reply:
I changed it to reference of unique_ptr since I'm allocating  new object in this function.